sdk/storage/azure-storage-blob/azure/storage/blob/aio/_models.py

```diff
@@ -161,6 +161,74 @@ def _build_item(self, item):
         return item
 
 
+class FilteredBlobPaged(AsyncPageIterator):
+    """An Iterable of Blob properties.
+
+    :ivar str service_endpoint: The service URL.
+    :ivar str prefix: A blob name prefix being used to filter the list.
+    :ivar str marker: The continuation token of the current page of results.
+    :ivar int results_per_page: The maximum number of results retrieved per API call.
+    :ivar str continuation_token: The continuation token to retrieve the next page of results.
+    :ivar str location_mode: The location mode being used to list results. The available
+        options include "primary" and "secondary".
+    :ivar current_page: The current page of listed results.
+    :vartype current_page: list(~azure.storage.blob.BlobProperties)
```

Comment:
Is this correct? I wasn't expecting a list to be named as "current_page"

Reply:
When filter blobs by page, current_page makes sense here..
It's consistent with BlobPropertiesPaged.